test.py

```diff
@@ -0,0 +1,70 @@
+import json, unittest
+
+# Create your tests here.
+from django.http import response
+from django.test import TestCase, Client, client
+
+from products.models import *
```

Comment:
```suggestion
from products.models import Category, SubCategoy....
```
실제 코드에서는 *을 지양해 주세요.

Reply:
넵 수정하겠습니다.